lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -283,37 +289,47 @@ defmodule ExUnit.DocTest do
   defp gen_code_for_tests(tests, whole_expr, stack) do
     quote do
       stack = unquote(stack)
+
       try do
         # Put all tests into one context
-        unquote_splicing(tests)
+        (unquote_splicing(tests))
```

Comment:
is this a bug?

Reply:
I will fix this one in the formatter, no need to worry about it for now.